helm/data-pipeline/ingress-nginx/mlab-sandbox.yml

```diff
@@ -0,0 +1,5 @@
+controller:
+  service:
+    loadBalancerIP: 34.27.69.115
```

Comment:
Was this IP address reserved manually by you? All new infrastructure should be deployed by Terraform.  Could we add this IP definition to the data-pipeline module in terraform-support?

https://github.com/m-lab/terraform-support/tree/main/modules/data-pipeline

Maybe add it to the data-pipeline.tf file?

Reply:
I'll open a separate PR in terraform-support. Would you prefer holding on approving this one until we have this IP (and the ones I need to add to staging and prod) fully managed by terraform?